README.md

```diff
@@ -1,87 +1,161 @@
+    - The **title** of your proposal should be the title of your tutorial. Take a minute to think of what would attract readers to view your tutorial. Keep it short and sweet! *Examples*: “Build a payment dApp on Solana” - “Polygon p2p payment dApp” - “Ceramic schema cookbook”.
+    - The **tags** field is useful to categorize your tutorial, and up to 5 can be selected.
+    Only select tags that are relevant to the topic of your proposed tutorial. Users will filter content on Kafé using these tags.
+    - Select a **difficulty level** for your tutorial, either Beginner or Experienced. If there is a lot of code involved in your tutorial, or it contains advanced concepts, Experienced is the better choice. This is also used to filter content when users are looking at the list of available tutorials.
+    - The **summary** of your proposal will be used as the description of the tutorial. The summary is limited to 480 characters, which is approximately 70 to 120 words including spaces. This is also used as a description when you generate the template to begin writing your tutorial.
+- When all the fields are complete, click on “submit proposal” on the right side of the page.
 
-### Quick start for authors.
+## Writing a tutorial for Kafé
 
-```bash
-yarn
-yarn build:cli
-builderdao --help
-```
-### How to start a tutorial.
+In order to prepare the tutorial directory, metadata and markdown template for your proposal, run the command:
+
+```text
+builderdao tutorial init
 ```
+Complete the interactive prompts to prepare the tutorial template and checkout the correct branch.
 
+First, you will need to select your tutorial by its **slug** which will match the title you gave your proposal, in lower-kebab-case. You can scroll through the list with the up and down arrow keys on your keyboard.
 
-```
+> For example, if your proposal title was “HTML 101”, you would select the slug `html-101`. 
 
+![](./assets/tutorial_init_1.png)
 
+- “Are you sure you want to create a tutorial for `...` ?” - **Answer “y” for Yes**.
+- “Are you sure you want to checkout the branch "`tutorials/<SLUG>`”? - **Answer "y" for Yes**.
+- Select tutorial type (choose **single** or **multipage**)
+- Tutorial Title - You can change the title of your tutorial _or_ press Enter to accept the Proposal title.
+- Tutorial Description - You can change the description of your tutorial _or_ press Enter to accept the Proposal description.
+- Select the keywords you would like to use (maximum of 5) - This list can be scrolled with the arrow keys, but unless you want to add any tags here it will already have the tags you added when submitting the proposal so you can press Enter to continue. 
+- Push changes? - **Answer "y" for Yes**.
 
+When this is complete, a new directory will be created for your tutorial inside the `tutorials` directory, matching the tutorial slug.
 
-## Writing a tutorial
+Inside, you will find:
```

Comment:
In this section, we need to be more explicit on where the author should go to write. There are a lot of files here, and we shouldn't assume that every author will know where to go to start writing. Also, if there are any files they shouldn't touch, we should be explicit with that as well.